docs/proposals/202012_receive_split.md

```diff
@@ -8,30 +8,79 @@ status: accepted
 ### Related Tickets
 
 * Hashring improvements: https://github.com/thanos-io/thanos/issues/3141
+* Previous proposal implementations: https://github.com/thanos-io/thanos/pull/3845, https://github.com/thanos-io/thanos/pull/3580
+* Distribution idea: https://github.com/thanos-io/thanos/pull/2675
 
 ### Summary
 
-This document describes the motivation and design of splitting the receive component into receive-route and receive.
-Additionally, we touch possibility for adding a consistent hashing mechanism and buffering the incoming requests.
+This document describes the motivation and design of running Receive in a stateless mode that does not have capabilities to store samples, it only routes remote write
+to further receives based on hashring. This allows setting optional deployment model were only Routing Receives are using hashring files and does the routing and replication. That allows ingesting Receivers to not handle any routing or hashring, only receiving multi tenant writes.
```

Comment:
Should it be `receives` or `receiver`? I am getting confused :(
```suggestion
to another receiver based on hashring. This allows setting optional deployment model were only Routing Receives are using hashring files and does the routing and replication. That allows ingesting Receivers to not handle any routing or hashring, only receiving multi tenant writes.
```

Reply:
Because out here in this line - `That allows ingesting Receivers to not handle any routing or hashring, only receiving multi tenant writes.`, we use `receiver` :)

Reply:
You are right: It should be "Receiver": https://thanos.io/tip/contributing/contributing.md/#components-naming